server/src/main/java/org/cloudfoundry/identity/uaa/account/ResetPasswordController.java

```diff
@@ -64,8 +64,9 @@ public String forgotPasswordPage(Model model,
                                      @RequestParam(required = false, value = "client_id") String clientId,
                                      @RequestParam(required = false, value = "redirect_uri") String redirectUri,
                                      HttpServletResponse response) {
-        if (!IdentityZoneHolder.get().getConfig().getLinks().getSelfService().isSelfServiceLinksEnabled()) {
-            return handleSelfServiceDisabled(model, response, "error_message_code", "self_service_disabled");
+        if (!IdentityZoneHolder.get().getConfig().getLinks().getSelfService().isSelfServiceResetPasswordEnabled()) {
```

Comment:
Create a variable for IdentityZoneHolder.get().getConfig().getLinks().getSelfService() for easier reading

Reply:
Done.